src/transactions/test/TxResultsTests.cpp

```diff
@@ -89,7 +89,7 @@ variations(int count, std::vector<T> values)
     return result;
 }
 
-TEST_CASE("txresults", "[tx][txresults]")
+TEST_CASE("txresults - errors", "[tx][txresults]")
```

Comment:
due to limitations in the parallel test scripts, we cannot use `-` in test names, only `[A-Za-z0-9 _()]+`